public/form_validation.js

```diff
@@ -0,0 +1,43 @@
+let elements = document.querySelectorAll('p.required');
+let inputs = document.querySelectorAll('p.required>input');
+let selects = document.querySelectorAll('p.required>select');
+            
+for(let i=0; i<elements.length; i++){
+        let el = elements[i];
+        let input = el.querySelector('input');
+        if(input){
+        input.onblur = function() {        
+        if (input.value === "") {
+        input.style.border = "1px solid red";
+        el.querySelector('span.error_text').style.display = "block";
+        el.querySelector('label').style.color="red";
+        }
+        };
+
+    input.onfocus = function() {
```

Comment:
@Varannchik , please use event listener instead of event handlers https://medium.com/geekculture/event-handlers-vs-event-listeners-in-javascript-b4086b8040b0, it is good general practice

Reply:
add event listener don't  work in this case